Packs/Inventa/Integrations/Inventa/Inventa.yml

```diff
@@ -0,0 +1,254 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Inventa
+  version: -1
+configuration:
+- defaultvalue: https://
```

Comment:
I think it would be more user-friendly to keep as is, as it's obvious that this field requires some valid URL for Inventa instance, yet it will not require to remove the placeholdler prior to entering the valid value.

Reply:
Ok sounds ok, take a look at the other comments